migrate/migrations/20210309144821-rotation-module-update.sql

```diff
@@ -0,0 +1,20 @@
+-- +migrate Up
+
+UPDATE engine_processing_versions
+SET version = 2
+WHERE type_id = 'rotation';
+
+ALTER TABLE rotation_state
+ADD COLUMN version INT DEFAULT 1 NOT NULL;
+
+ALTER TABLE rotation_state
+ALTER COLUMN version SET DEFAULT 2;
```

Comment:
Are these redundant since they are in the same migration? As in, would it make more sense to only do `DEFAULT 2 NOT NULL`?

Reply:
Adding the column with a default will set all existing/old rows to version 1, then we change the default to 2 so that any new rows start at version 2. That lets us migrate just the existing data

Reply:
Gotcha, I comprehended it as it setting the default to 1 for all existing rows, but then immediately after changing all of them to 2. This makes more sense 👍